packages/components/Button/src/ExtractStyle.android.ts

```diff
@@ -0,0 +1,57 @@
+import { memoize } from '@fluentui-react-native/framework';
+import { ViewStyle } from 'react-native';
+
+/**
+ * React Native's Pressable does not support curved edges.
+ * It needs to be wrapped inside another view and have border set there.
+ * This function extracts styles that should be applied on the outer view.
+ *
+ * @param style Styling that is to be applied on the component
+ * @returns Array containing split styles that are to be applied on the inner and outer views
+ */
+
+export const extractOuterStylePropsAndroid = memoize((style: ViewStyle = {}): [outerStyleProps: ViewStyle, innerStyleProps: ViewStyle] => {
+  const {
+    margin,
+    marginTop,
+    marginBottom,
+    marginLeft,
+    marginRight,
+    marginStart,
+    marginEnd,
+    marginVertical,
+    marginHorizontal,
+    start,
+    end,
+    left,
+    right,
+    top,
+    bottom,
+    display,
+    opacity,
+    ...restOfProps
+  } = style;
+
+  return [
+    {
+      margin,
+      marginTop,
+      marginBottom,
+      marginLeft,
+      marginRight,
+      marginStart,
+      marginEnd,
+      marginVertical,
+      marginHorizontal,
+      start,
+      end,
+      left,
+      right,
+      top,
+      bottom,
+      display,
+      opacity,
+    },
+    { borderWidth: 0, ...restOfProps },
```

Comment:
Why is borderWidth: 0? I'm guessing to account for the ripple still?

Reply:
To ensure inside view doesn't have a border width, had initially thought of doing this in the styling, but have placed all the code related to ripple here in one place